src/store.js

```diff
@@ -1,10 +1,7 @@
 import { createContext } from "react";
+import { makeBoard } from "./game-mechanics";
 
-export const GAME_STATE_INIT = [
```

Comment:
The goals of this was to also act as a visualization of the game board itself.

Reply:
I can add a comment with the board shape if that helps?